docs/api/dataset.rst

```diff
@@ -0,0 +1,99 @@
+Datasets
+========
+
+General datasets
+----------------
+
+.. automodule:: fuel.datasets.base
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+Adult dataset
+-------------
+
+.. automodule:: fuel.datasets.adult
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+billion dataset
```

Comment:
I'd replace this title with `One Billion Word` (which is the name of the benchmark).